src/work-allocation-2/containers/task-assignment-choose-role/task-assignment-choose-role.component.ts

```diff
@@ -104,6 +104,7 @@ export class TaskAssignmentChooseRoleComponent implements OnInit {
   }
 
   private userWithOwnPermission(roles: TaskRole[]): TaskRole {
-    return roles.find(role => role.permissions.includes(Permissions.Own));
+    // EUI-5236 - TaskPermission instead of Permission (i.e. not all caps)
+    return roles.find(role => role.permissions.includes(TaskPermission.OWN));
```

Comment:
@connorpgpmcelroy  I think we should look for both the cases. With case sensitive on/off